openupgrade_framework/readme/CREDITS.md

```diff
@@ -0,0 +1,4 @@
+Many developers have contributed to the OpenUpgrade framework in its
+previous incarnation. Their original contributions may no longer needed,
```

Comment:
s/many no longer needed/many no longer be needed/

Reply:
are you sure it can be used? It adds extra commits noise that many dislike. I don't see the feature used anymore in the OCA. Can you confirm it's easy enough to remove the commit noise vs doing it this way?

Reply:
I don't use the github feature for committing myself, because indeed it creates extra commits I'd have to squash again. But still I appreciate reviewers using the feature, because that makes it perfectly clear what they mean.

Reply:
>  But still I appreciate reviewers using the feature, because that makes it perfectly clear what they mean.

:+1: That is my point. Regex expression is not very easy to read.

> I don't use the github feature for committing myself, because indeed it creates extra commits I'd have to squash again

Matter of taste. Once the change proposal is done you can : 
- merge then squash
- or make changes locally and push